model/labels/regexp.go

```diff
@@ -767,7 +768,7 @@ type equalMultiStringMapMatcher struct {
 
 func (m *equalMultiStringMapMatcher) add(s string) {
 	if !m.caseSensitive {
-		s = strings.ToLower(s)
+		s = strings.ToLower(norm.NFKD.String(s))
```

Comment:
Now thank you for nerd-sniping me on this.

I tried [reading the docs](https://unicode.org/reports/tr15/#Norm_Forms) and even though I'd have chosen `NFKC` for this, I ran the tests in your branch, especially with the example character from the docs `ẛ̣` and I couldn't tell any difference (in test results) between both normalizations, so LGTM.